src/analysis/analysis-model.js

```diff
@@ -110,6 +122,10 @@ var AnalysisModel = Model.extend({
     return this._anyStatus(STATUS.READY, STATUS.FAILED);
```

Comment:
`_anyStatus` doesnt look a very descriptive method name to me, we could just use `_.contains`

```javascript
  _.contains([STATUS.READY, STATUS.FAILED], this.get('status'));

```